_posts/2018-11-16-webhooks-companies.markdown

```diff
@@ -0,0 +1,130 @@
+---
+permalink: v1/webhooks/companies
+title: Companies Webhooks
+layout: sidebar
+---
+
+# Companies Webhooks
+
+## Get Company Webhook Notifications
+
+[Webhooks Overview](/v1/webhooks/about)
+
+**Resource**: `Company`
+
+**Entity**: `Company` ([API Reference](/v1/companies))
+
+
+**Event Types**:
+
+- `company.provisioned`
+
+- `company.updated`
+
+- `company.deprovisioned`
+
+**Returns**: Get information about a particular company that was created or updated
+
+```json
+{
+  "event_type": "company.provisioned",
+  "resource_type": "Company",
+  "resource_id": 7757500908984137,
+  "entity_type": "Company",
+  "entity_id": 7757500908984137,
+  "timestamp": 1533606328,
+  "entity_attributes": {
+    "id": 7757500908984137,
```

Comment:
wonder if jekyll supports partials to avoid these webhook payloads getting out of sync with the resource doc.

Not sure if this is it https://jekyllrb.com/docs/includes/

Reply:
I think they do. I see the usefulness but these payloads should be dynamically generated from our serializers/presenters/decorators right in the public API if we _really_ cared about having the docs stay consisten 🤷‍♂️ 

If you feel this has a useful ROI I can give it a shot.

Reply:
Actually pulling each resource out of the resource docs so that they can be referenced from the `_includes` directory seems to add enough that it is out of scope. The scope further increases if we consider doing the "right thing" where we pull out _all_ of the other payloads and not just the ones we need for the webhooks sharing.

Reply:
I agree the payload should be generated, but I think when a dev working on a change to the resource api they may not remember to update webhook api or even aware of it. Forcing the payloads to share the same file makes sure the doc is updated in both places, thoughts?

Reply:
Can you please open a separate ticket to track this then. Today is your last day with Partners? 😭